hack/run-promoted-image-governor.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+cd $(dirname $0)/..
+
+kubeconfig="$(mktemp)"
+
+trap 'rm -f $kubeconfig' EXIT
+oc --context app.ci --as system:admin --namespace ci serviceaccounts create-kubeconfig promoted-image-governor >$kubeconfig
```

Comment:
IMO we can have this as a make target like the other ones for e.g. local-e2e. no need to make people re-download this every time

Reply:
Updated.